ci/cfg/ci_simple_capture_aes_sca.yaml

```diff
@@ -0,0 +1,40 @@
+cwfpgahusky:
```

Comment:
Can you please create an issue to check whether we can unify the ci configs with the default configs currently living under cw/? I think it would save us quite some maintenance work by not having to continuously adjust two configs files.

Reply:
Done: #209.